lib/livebook_web/live/env_var_component.ex

```diff
@@ -21,21 +21,28 @@ defmodule LivebookWeb.SettingsLive.EnvVarComponent do
 
   @impl true
   def render(assigns) do
+    assigns =
+      assigns
+      |> assign_new(:on_save, fn -> "save" end)
+      |> assign_new(:resource, fn -> "environment variable" end)
```

Comment:
I think we don't need an assign for this?

Reply:
Yeah, we don't need anymore